keyboards/basekeys/slice/keymaps/default/keymap.c

```diff
@@ -0,0 +1,161 @@
+#include QMK_KEYBOARD_H
+
+extern uint8_t is_master;
+
+#ifdef EE_HANDS
+// OLED_ROTATION_180 for left
+// OLED_ROTATION_0 for right
+#define screen OLED_ROTATION_180
+#endif
+
```

Comment:
```suggestion
```

Reply:
3 Keymaps were updated with this change.

keymaps/default/keymap.c
keymaps/default_split_left_space/keymap.c
rev1_rgb/keymaps/slice_rgb/keymap.c

Reply:
3 Keymaps were updated with this change.

keymaps/default/keymap.c
keymaps/default_split_left_space/keymap.c
rev1_rgb/keymaps/slice_rgb/keymap.c